tests/e2e/upgrade_test.go

```diff
@@ -191,8 +191,15 @@ var _ = Describe("Upgrade", Label(tests.LabelUpgrade, tests.LabelNoOpenshift), O
 				Eventually(func() (int, error, error) {
 					stdout, _, err := env.ExecCommand(env.Ctx, pod, specs.PostgresContainerName, &commandTimeout,
 						"psql", "-U", "postgres", "-tAc", "show max_replication_slots")
-					value, atoiErr := strconv.Atoi(strings.Trim(stdout, "\n"))
-					return value, err, atoiErr
+					if err != nil {
+						return 0, err, nil
+					}
```

Comment:
The whole point of having multiple errors in the return is avoiding explicitly checking the errors.

If we handle the errors, we can return one.